content/en/docs/restaurant/how-to/kds_setup/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Set up KDS"
+description: "There are several administrative sections that need to be configured in Business Central before you can effectively use KDS (Kitchen Display System)."
+lead: ""
+date: 2024-04-11T10:46:53+02:00
+lastmod: 2024-04-11T10:46:53+02:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: ""
+    identifier: "kds_setup-027de2bef2455b4cdd53b139c1867a7b"
+weight: 341
+toc: true
+type: docs
+---
+
+There are several administrative sections that need to be configured in Business Central before you can effectively use KDS (Kitchen Display System). 
+
+To set up prerequisites and the KDS, follow the provided steps:
+
+#### Prerequisites
+
+- Create and fully configure **Item Cards** which correspond to dishes that are going to be prepared in the restaurant.
+- Add at least one restaurant to the environment, and create a seating plan.
+- Create a **Restaurant Profile**, and assign it to POS units.
+
+#### Procedure 
```

Comment:
I'm not sure what the purpose of this document is. If it is just to describe the steps required specifically for KDS, then steps 3 to 12 in the procedure are not relevant, as these should be set up regardless of whether you have KDS or not (as an alternative to KDS, we can use kitchen printers, which also provide information to the kitchen about products to be prepared).
We can leave it as it is for now, and rearrange the articles later, when the full restaurant documentation is ready.